code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -34,124 +31,257 @@
 
-	apply_cooldown()
+	if (!action_cooldown_check())
+		if(twitch_message_cooldown < world.time )
+			predalien_smash.visible_message(SPAN_XENOWARNING("[predalien_smash]'s muscles twitch."), SPAN_XENOWARNING("Our claws twitch as we try to grab onto the target but lack the strength. Wait a moment to try again."))
+			twitch_message_cooldown = world.time + 5 SECONDS
+		return //this gives a little feedback on why your lunge didn't hit other than the lunge button going grey. Plus, it might spook marines that almost got lunged if they know why the message appeared, and extra spookiness is always good.
 
-	ADD_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
-	xeno.anchored = TRUE
+	if (!affected_atom)
+		return
 
-	if (do_after(xeno, activation_delay, INTERRUPT_ALL | BEHAVIOR_IMMOBILE, BUSY_ICON_HOSTILE))
-		xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] rips open the guts of [carbon]!"), SPAN_XENOHIGHDANGER("We rip open the guts of [carbon]!"))
-		carbon.spawn_gibs()
-		playsound(get_turf(carbon), 'sound/effects/gibbed.ogg', 75, 1)
-		carbon.apply_effect(get_xeno_stun_duration(carbon, 0.5), WEAKEN)
-		carbon.apply_armoured_damage(get_xeno_damage_slash(carbon, base_damage + damage_scale * behavior.kills), ARMOR_MELEE, BRUTE, "chest", 20)
+	if (!isturf(predalien_smash.loc))
+		to_chat(predalien_smash, SPAN_XENOWARNING("We can't lunge from here!"))
+		return
 
-		xeno.animation_attack_on(carbon)
-		xeno.spin_circle()
-		xeno.flick_attack_overlay(carbon, "tail")
+	if (!predalien_smash.check_state() || predalien_smash.agility)
+		return
 
-	playsound(owner, 'sound/voice/predalien_growl.ogg', 75, 0, status = 0)
+	if(predalien_smash.can_not_harm(affected_atom) || !ismob(affected_atom))
+		apply_cooldown_override(click_miss_cooldown)
+		return
 
-	REMOVE_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
-	xeno.anchored = FALSE
-	unroot_human(carbon, TRAIT_SOURCE_ABILITY("Devastate"))
+	var/mob/living/carbon/carbon = affected_atom
+	if(carbon.stat == DEAD)
+		return
+
+	if (!check_and_use_plasma_owner())
+		return
 
-	xeno.visible_message(SPAN_XENODANGER("[xeno] rapidly slices into [carbon]!"))
+	apply_cooldown()
 
+	predalien_smash.throw_atom(get_step_towards(affected_atom, predalien_smash), grab_range, SPEED_FAST, predalien_smash)
+
+	if (predalien_smash.Adjacent(carbon))
+		predalien_smash.start_pulling(carbon,1)
+		if(isliving(carbon))
+			playsound(carbon.pulledby, 'sound/voice/predalien_growl.ogg', 75, 0, status = 0) // bang and roar for dramatic effect
+			playsound((carbon), 'sound/effects/bang.ogg', 25, 0)
+			animate(carbon, pixel_y = carbon.pixel_y + 32, time = 4, easing = SINE_EASING)
+			sleep(4)
+			playsound((carbon), 'sound/effects/bang.ogg', 25, 0) // bang for dramatic damage effect
+			playsound((carbon),"slam", 50, 1)
+			animate(carbon, pixel_y = 0, time = 4, easing = BOUNCE_EASING)
+			var/turf/back_to_middle = get_step(carbon, SOUTH) // move them back one tile south
+			carbon.forceMove(back_to_middle)
```

Comment:
```suggestion
```
We aren't moving them; we're animating their pixel_y. If you grabbed them when you are north of them, the move southward will break your grip.